tests/linux_kernel/helpers/test_slab.py

```diff
@@ -75,6 +77,37 @@ def test_slab_cache_is_merged_true(self):
             self.fail("couldn't find slab cache")
         self.assertTrue(slab_cache_is_merged(slab_cache))
 
+    def test_get_slab_cache_aliases(self):
+        if not SLAB_SYSFS_PATH.exists():
+            # A SLOB or SLAB kernel, or one without SYSFS. Test that the
+            # helper fails as expected.
+            self.assertRaises(LookupError, get_slab_cache_aliases, self.prog)
```

Comment:
I think it's valuable to sanity check that we caught the right error and not some random unrelated error, so how about we at least do something like:

```suggestion
            self.assertRaisesRegex(LookupError, "CONFIG_SYSFS", get_slab_cache_aliases, self.prog)
```